interfaces/builtin/desktop_test.go

```diff
@@ -117,11 +118,10 @@ func (s *DesktopInterfaceSuite) TestAppArmorSpec(c *C) {
 	spec = &apparmor.Specification{}
 	c.Assert(spec.AddConnectedPlug(s.iface, s.plug, s.coreSlot), IsNil)
 	updateNS = spec.UpdateNS()
-	c.Assert(updateNS, HasLen, 4)
-	c.Check(updateNS[0], testutil.Contains, "# Mount the document portal")
-	c.Check(updateNS[1], testutil.Contains, "# Read-only access to /usr/share/fonts")
-	c.Check(updateNS[2], testutil.Contains, "# Read-only access to /usr/local/share/fonts")
-	c.Check(updateNS[3], testutil.Contains, "# Read-only access to /var/cache/fontconfig")
+	c.Check(updateNS, testutil.Contains, "  # Mount the document portal\n")
+	c.Check(updateNS, testutil.Contains, "  # Read-only access to /usr/share/fonts\n")
+	c.Check(updateNS, testutil.Contains, "  # Read-only access to /usr/local/share/fonts\n")
+	c.Check(updateNS, testutil.Contains, "  # Read-only access to /var/cache/fontconfig\n")
```

Comment:
same question as above, is it still useful to check the length of the UpdateNS snippet?